timeseries/src/main/scala/com/criteo/cuttle/timeseries/TimeSeriesScheduler.scala

```diff
@@ -150,11 +150,11 @@ case class TimeSeriesScheduler() extends Scheduler[TimeSeries] with TimeSeriesAp
         _backfills += newBackfill
         _state() = _state() ++ jobs.map((job: TimeSeriesJob) =>
           job -> (_state().apply(job) - Interval.closedOpen(start, end)))
+        Database.createBackfill(newBackfill).transact(xa).unsafePerformIO
```

Comment:
You can't do IO inside an STM transaction. Just use `result` like
`result.right.foreach (_ => Database.createBackfill(newBackfill).transact(xa).unsafePerformIO)`